paket.lock

```diff
@@ -613,692 +613,608 @@ NUGET
+    Microsoft.Win32.Registry (5.0) - restriction: || (&& (== net472) (< net45)) (&& (== net472) (< net462)) (&& (== net472) (>= netstandard2.1)) (== net6.0) (== net8.0) (== netcoreapp2.1) (== netstandard2.0)
+      System.Buffers (>= 4.5.1) - restriction: || (&& (== net472) (== net6.0)) (&& (== net472) (== net8.0)) (&& (== net472) (== netcoreapp2.1)) (&& (== net472) (>= monoandroid) (< netstandard1.3)) (&& (== net472) (>= monotouch)) (&& (== net472) (< net46)) (&& (== net472) (>= xamarinios)) (&& (== net472) (>= xamarinmac)) (&& (== net472) (>= xamarintvos)) (&& (== net472) (>= xamarinwatchos)) (&& (== net6.0) (>= monoandroid) (< netstandard1.3)) (&& (== net6.0) (>= monotouch)) (&& (== net6.0) (< netcoreapp2.0)) (&& (== net6.0) (>= xamarinios)) (&& (== net6.0) (>= xamarinmac)) (&& (== net6.0) (>= xamarintvos)) (&& (== net6.0) (>= xamarinwatchos)) (&& (== net8.0) (>= monoandroid) (< netstandard1.3)) (&& (== net8.0) (>= monotouch)) (&& (== net8.0) (< netcoreapp2.0)) (&& (== net8.0) (>= xamarinios)) (&& (== net8.0) (>= xamarinmac)) (&& (== net8.0) (>= xamarintvos)) (&& (== net8.0) (>= xamarinwatchos)) (&& (== netcoreapp2.1) (>= monoandroid) (< netstandard1.3)) (&& (== netcoreapp2.1) (>= monotouch)) (&& (== netcoreapp2.1) (< netcoreapp2.0)) (&& (== netcoreapp2.1) (>= xamarinios)) (&& (== netcoreapp2.1) (>= xamarinmac)) (&& (== netcoreapp2.1) (>= xamarintvos)) (&& (== netcoreapp2.1) (>= xamarinwatchos)) (== netstandard2.0)
+      System.Memory (>= 4.5.4) - restriction: || (&& (== net472) (== net6.0)) (&& (== net472) (== net8.0)) (&& (== net472) (== netcoreapp2.1)) (&& (== net472) (< net46)) (&& (== net472) (>= netcoreapp2.0)) (&& (== net472) (>= uap10.1)) (&& (== net6.0) (< netcoreapp2.0)) (&& (== net6.0) (< netcoreapp2.1)) (&& (== net6.0) (>= uap10.1)) (&& (== net8.0) (< netcoreapp2.0)) (&& (== net8.0) (< netcoreapp2.1)) (&& (== net8.0) (>= uap10.1)) (&& (== netcoreapp2.1) (< netcoreapp2.0)) (&& (== netcoreapp2.1) (>= uap10.1)) (== netstandard2.0)
       System.Security.AccessControl (>= 5.0)
       System.Security.Principal.Windows (>= 5.0)
-    Microsoft.Win32.SystemEvents (6.0) - restriction: || (== net6.0) (&& (== netcoreapp2.1) (>= netcoreapp3.1)) (&& (== netstandard2.0) (>= netcoreapp3.1))
+    Microsoft.Win32.SystemEvents (6.0) - restriction: || (&& (== net472) (>= net6.0)) (&& (== net472) (>= netcoreapp3.1)) (== net6.0) (== net8.0) (&& (== netcoreapp2.1) (>= netcoreapp3.1)) (&& (== netstandard2.0) (>= netcoreapp3.1))
     Mono.Cecil (0.11.5)
     Mono.Posix.NETStandard (1.0)
-    MSBuild.StructuredLogger (2.1.815)
-      Microsoft.Build.Framework (>= 16.10)
-      Microsoft.Build.Utilities.Core (>= 16.10)
+    MSBuild.StructuredLogger (2.2.317)
+      DotUtils.StreamUtils.Sources (>= 0.0.7)
+      Microsoft.Build.Framework (>= 17.5)
+      Microsoft.Build.Utilities.Core (>= 17.5)
+      Microsoft.SourceLink.GitHub (>= 8.0)
```

Comment:
fwiw I'm wondering if the latest StructuredLogger package [might have accidentaly made some dev dependencies into runtime dependencies](https://github.com/KirillOsenkov/MSBuildStructuredLog/pull/798#discussion_r1742775769) - maybe some of these new deps might not really be needed anyway

Reply:
I fixed this in https://www.nuget.org/packages/MSBuild.StructuredLogger/2.2.337